### ISSUE TITLE
Keras learning rate recorded incorrectly when decay/scheduling callbacks used

### ISSUE THREAD

**Issue by HunterMcGushion** [HunterMcGushion/hyperparameter_hunter#50]
- See :meth:`recorders.DescriptionRecorder.format_result`
- Model's `get_config()` returns the final learning rate, rather than the initial one, so experiment description files are misleading by not displaying the actual value used
- Leads to failed similar experiment matches
- Experiment started with Adam at lr=0.001, and `ReduceLROnPlateau`, which dropped the lr down to 0.0001
   - 0.0001 was recorded as the Experiment's lr, but it should be 0.001
- Probably need to call `parameterize_compiled_keras_model` immediately after initializing it, then store the results, then use them in the `DescriptionRecorder`
   - Midway through `experiments.BaseCVExperiment.cv_run_workflow`, or in `models.KerasModel.initialize_model/fit`
